diff -Nurp old/domainjoin/libdomainjoin/src/djdistroinfo.c new/domainjoin/libdomainjoin/src/djdistroinfo.c
--- old/domainjoin/libdomainjoin/src/djdistroinfo.c	2009-11-03 10:53:44.000000000 -0800
+++ new/domainjoin/libdomainjoin/src/djdistroinfo.c	2009-11-03 17:02:38.000000000 -0800
@@ -624,7 +624,7 @@ CENTERROR DJGetLikewiseVersion(PSTR *ver
 #ifdef MINIMAL_JOIN
     GCE(ceError = CTOpenFile(LOCALSTATEDIR "/VERSION", "r", &versionFile));
 #else
-    GCE(ceError = CTOpenFile(PREFIXDIR "/data/VERSION", "r", &versionFile));
+    GCE(ceError = CTOpenFile(PREFIXDIR "/share/likewise-open/VERSION", "r", &versionFile));
 #endif
 
     while (TRUE)

