Index: likewise-open-5.4.0.41804/domainjoin/domainjoin-cli/src/main.c
===================================================================
--- likewise-open-5.4.0.41804.orig/domainjoin/domainjoin-cli/src/main.c	2010-03-05 15:45:28.000000000 -0600
+++ likewise-open-5.4.0.41804/domainjoin/domainjoin-cli/src/main.c	2010-03-05 15:45:58.000000000 -0600
@@ -478,6 +478,7 @@
 
     LW_TRY(exc, DJRunJoinProcess(&options, &LW_EXC));
     fprintf(stdout, "SUCCESS\n");
+    fprintf(stdout, "You should reboot this system before attempting GUI logins as a domain user.\n");
 
 cleanup:
     DJFreeJoinProcessOptions(&options);
