Index: likewise-open-5.4.0.41804/domainjoin/domainjoin-gui/gtk/common.c
===================================================================
--- likewise-open-5.4.0.41804.orig/domainjoin/domainjoin-gui/gtk/common.c	2010-03-05 15:49:50.000000000 -0600
+++ likewise-open-5.4.0.41804/domainjoin/domainjoin-gui/gtk/common.c	2010-03-05 15:49:56.000000000 -0600
@@ -33,7 +33,7 @@
 void
 dialog_insert_likewise_logo(GtkDialog* dialog)
 {
-    GtkWidget* image = gtk_image_new_from_file (DATADIR "/likewise/likewise-logo.png");
+    GtkWidget* image = gtk_image_new_from_file (DATADIR "/likewise-open/likewise-logo.png");
 
     if (image)
     {
Index: likewise-open-5.4.0.41804/domainjoin/domainjoin-gui/gtk/common.h
===================================================================
--- likewise-open-5.4.0.41804.orig/domainjoin/domainjoin-gui/gtk/common.h	2010-03-05 15:49:50.000000000 -0600
+++ likewise-open-5.4.0.41804/domainjoin/domainjoin-gui/gtk/common.h	2010-03-05 15:49:56.000000000 -0600
@@ -34,7 +34,7 @@
 #include <glade/glade.h>
 #include <gtk/gtk.h>
 
-#define DOMAINJOIN_XML (DATADIR "/likewise/domainjoin-gtk.glade")
+#define DOMAINJOIN_XML (DATADIR "/likewise-open/domainjoin-gtk.glade")
 
 typedef struct __JOINSTATE
 {
