diff -Nurb likewise-open/domainjoin/libdomainjoin/src/djnsswitch.c likewise-open.new/domainjoin/libdomainjoin/src/djnsswitch.c
--- likewise-open/domainjoin/libdomainjoin/src/djnsswitch.c	2009-11-12 11:38:04.000000000 -0800
+++ likewise-open.new/domainjoin/libdomainjoin/src/djnsswitch.c	2009-11-12 15:51:44.000000000 -0800
@@ -1058,14 +1058,14 @@
 PREFIXDIR "/lib/*.so*            mr,\n"
 PREFIXDIR "/lib64/*.so*          mr,\n"
 "/tmp/.lwidentity/pipe              rw,\n"
-LOCALSTATEDIR "/lib/likewise/.lsassd  rw,\n"
+LOCALSTATEDIR "/.lsassd  rw,\n"
 LOCALSTATEDIR "/tmp/.lsaclient_*              rw,\n";
     else
         addString =
 PREFIXDIR "/lib/*.so*            r,\n"
 PREFIXDIR "/lib64/*.so*          r,\n"
 "/tmp/.lwidentity/pipe              rw,\n"
-LOCALSTATEDIR "/lib/likewise/.lsassd  rw,\n"
+LOCALSTATEDIR "/.lsassd  rw,\n"
 LOCALSTATEDIR "/tmp/.lsaclient_*              rw,\n";
 
 
@@ -1073,7 +1073,7 @@
     {
         LW_CLEANUP_CTERR(exc, CTCopyFileWithOriginalPerms(finalName, tempName));
         LW_CLEANUP_CTERR(exc, CTOpenFile(tempName, "a", &file));
-        LW_CLEANUP_CTERR(exc, CTFilePrintf(file, "# likewise\n%s# end likewise\n",
+        LW_CLEANUP_CTERR(exc, CTFilePrintf(file, "# -start- likewise\n%s# -end- likewise\n",
                     addString));
 
         CTSafeCloseFile(&file);
@@ -1082,7 +1082,7 @@
     }
     else
     {
-        LW_CLEANUP_CTERR(exc, CTRunSedOnFile(finalName, finalName, FALSE, "/^[ \t]*#[ \t]*likewise[ \t]*$/,/^[ \t]*#[ \t]*end likewise[ \t]*$/d"));
+        LW_CLEANUP_CTERR(exc, CTRunSedOnFile(finalName, finalName, FALSE, "/^[ \t]*#[ \t]*-start- likewise[ \t]*$/,/^[ \t]*#[ \t]*-end- likewise[ \t]*$/d"));
         LW_CLEANUP_CTERR(exc, CTRunSedOnFile(finalName, finalName, FALSE, "/^[ \t]*#[ \t]*centeris[ \t]*$/,/^[ \t]*#[ \t]*end centeris[ \t]*$/d"));
     }
 
