Index: likewise-open-5.4.0.41804/build/components/krb5.comp
===================================================================
--- likewise-open-5.4.0.41804.orig/build/components/krb5.comp	2010-03-05 13:04:08.000000000 -0600
+++ likewise-open-5.4.0.41804/build/components/krb5.comp	2010-03-05 15:51:29.000000000 -0600
@@ -94,7 +94,7 @@
     mkdir -m 0755 -p ${STAGING_PREFIX_DIR}/{bin,include,${_lib}}
     mkdir -m 0755 -p ${STAGING_SYSCONF_DIR}
 
-    for file in kinit klist kdestroy kpasswd kvno ksu krb5-config; do
+    for file in kinit klist kdestroy kpasswd kvno ksu krb5-config ktutil; do
         cp ${INSTALL_PREFIX_DIR}/bin/${file} ${STAGING_PREFIX_DIR}/bin/.
     done
 
