Index: likewise-open-4.0.4/domainjoin/scripts/ConfigureLogin.in
===================================================================
--- likewise-open-4.0.4.orig/domainjoin/scripts/ConfigureLogin.in	2008-02-11 15:50:50.000000000 -0600
+++ likewise-open-4.0.4/domainjoin/scripts/ConfigureLogin.in	2008-02-11 15:13:44.000000000 -0600
@@ -463,6 +463,13 @@
 
 ConfigureAppArmor()
 {
+    ## This is a no op on Ubuntu & Debian
+
+    local distro_type=`GetDistroType`
+    if [ "$distro_type" = "ubuntu" -o "$distro_type" = "debian" ]; then
+        return 0
+    fi
+
     if [ 'enable' = "$1" ]; then
 	if [ -f /etc/apparmor.d/abstractions/nameservice ]; then
 	    if grep "centeris" /etc/apparmor.d/abstractions/nameservice 2>&1 >/dev/null; then
@@ -472,21 +479,25 @@
 		if grep "mr," /etc/apparmor.d/abstractions/nameservice 2>&1 >/dev/null; then
 		    cat >>/etc/apparmor.d/abstractions/nameservice <<__AAEOF__
 # centeris
-/usr/centeris/lib/*.so*     mr,
-/usr/centeris/lib64/*.so*   mr,
+/usr/lib/libwbclient.so*     mr,
+/usr/lib64/libwbclient.so*   mr,
 /tmp/.lwidentity/pipe       rw,
 # end centeris
 __AAEOF__
 		else
 		    cat >>/etc/apparmor.d/abstractions/nameservice <<__AAEOF2__
 # centeris
-/usr/centeris/lib/*.so*     r,
-/usr/centeris/lib64/*.so*   r,
+/usr/lib/libwbclient.so*     r,
+/usr/lib64/libwbclient.so*   r,
 /tmp/.lwidentity/pipe       rw,
 # end centeris
 __AAEOF2__
 		fi
-		rcapparmor restart || return $?
+		if [ -x /etc/init.d/apparmor ]; then
+			/etc/init.d/apparmor reload || return $?
+		else
+			rcapparmor restart || return $?
+		fi
 	    fi
 	fi
     fi
