diff --git a/domainjoin/domainjoin-gui/src/main.c b/domainjoin/domainjoin-gui/src/main.c
index b94c08f..7794305 100644
--- likewise-4.0.4.orig/domainjoin/domainjoin-gui/src/main.c
+++ likewise-4.0.4/domainjoin/domainjoin-gui/src/main.c
@@ -324,7 +324,7 @@ do_join(JoinDialog* dialog, LWException** exc)
 	}
 
 	info.dialog = progress_dialog;
-	info.options.noModifyHosts = joindialog_get_modify_hosts(dialog);
+	info.options.noModifyHosts = !joindialog_get_modify_hosts(dialog);
 
 	g_thread_create(join_worker, &info, FALSE, NULL);
 
