diff --git a/source/utils/smbcontrol.c b/source/utils/smbcontrol.c
index 22593e4..2ae2e07 100644
--- likewise-4.0.4.orig/winbindd/source/utils/smbcontrol.c
+++ likewise-4.0.4/winbindd/source/utils/smbcontrol.c
@@ -1255,11 +1255,11 @@ static struct server_id parse_dest(const char *dest)
 	}
 
 	/* Fix winbind typo. */
-	if (strequal(dest, "winbind")) {
-		dest = "lwiauthd";
+	if (strequal(dest, "winbind") || strequal(dest, "winbindd")) {
+		dest = "likewise-winbindd";
 	}
 	
-	if (!(strequal(dest, "lwiauthd") || strequal(dest, "nmbd"))) {
+	if (!(strequal(dest, "likewise-winbindd") || strequal(dest, "nmbd"))) {
 		/* Check for numeric pid number */
 
 		result = interpret_pid(dest);
