Index: samba/source/winbindd/winbindd_cache.c
===================================================================
--- a/samba/source/winbindd/winbindd_cache.c	(revision 27109)
+++ b/samba/source/winbindd/winbindd_cache.c	(working copy)
@@ -3582,12 +3582,17 @@
 static int cache_traverse_validate_fn(TDB_CONTEXT *the_tdb, TDB_DATA kbuf, TDB_DATA dbuf, void *state)
 {
 	int i;
+	unsigned int max_key_len = 1024;
 	struct tdb_validation_status *v_state = (struct tdb_validation_status *)state;
 
 	/* Paranoia check. */
-	if (kbuf.dsize > 1024) {
-		DEBUG(0,("cache_traverse_validate_fn: key length too large (%u) > 1024\n\n",
-				(unsigned int)kbuf.dsize ));
+	if (strncmp("UA/", (const char *)kbuf.dptr, 3) == 0) {
+		max_key_len = 1024 * 1024;
+	}
+	if (kbuf.dsize > max_key_len) {
+		DEBUG(0, ("cache_traverse_validate_fn: key length too large: "
+			  "(%u) > (%u)\n\n",
+			  (unsigned int)kbuf.dsize, (unsigned int)max_key_len));
 		return 1;
 	}
 
