diff --git a/source/Makefile.in b/source/Makefile.in
index ee60c6c..3de28fa 100644
--- likewise-4.0.4.orig/winbindd/source/Makefile.in
+++ likewise-4.0.4/winbindd//source/Makefile.in
@@ -1056,35 +1056,30 @@ nss_modules : $(NSS_MODULES)
 ## Use hard links to prevent unnecessary rebuilds but not consume
 ## extra disk space
 
-bin/pam_lwidentity.@SHLIBEXT@: bin/pam_winbind.@SHLIBEXT@
-	@if test ! -f $@; then\
-		ln bin/pam_winbind.@SHLIBEXT@ $@;\
-	 fi
-
 @WINBIND_NSS_LWI@: @WINBIND_NSS@
-	@if test ! -f $@; then\
-		cp -f @WINBIND_NSS@  $@;\
-	 fi
+	@echo "Linking $@"
+	@$(SHLD) $(WINBIND_NSS_LDSHFLAGS) -o $@ $(WINBIND_NSS_OBJ) \
+		@WINBIND_NSS_EXTRA_LIBS@ @WINBIND_NSS_PTHREAD@ @SONAMEFLAG@`basename $@`@NSSSONAMEVERSIONSUFFIX@
+
+bin/pam_lwidentity.@SHLIBEXT@: bin/pam_winbind.@SHLIBEXT@
+	@if test -f $@; then /bin/rm $@; fi
+	@ln -f bin/pam_winbind.@SHLIBEXT@ $@
 
 bin/likewise-winbindd@EXEEXT@: bin/winbindd@EXEEXT@
-	@if test ! -f $@; then \
-		ln -f bin/winbindd@EXEEXT@ $@;\
-	 fi
+	@if test -f $@; then /bin/rm $@; fi
+	@ln -f bin/winbindd@EXEEXT@ $@
 
 bin/lwinet@EXEEXT@: bin/net@EXEEXT@
-	@if test ! -f $@; then \
-		ln -f bin/net@EXEEXT@ $@;\
-	 fi
+	@if test -f $@; then /bin/rm $@; fi
+	@ln -f bin/net@EXEEXT@ $@
 
 bin/lwiinfo@EXEEXT@: bin/wbinfo@EXEEXT@
-	@if test ! -f $@; then\
-		ln bin/wbinfo@EXEEXT@ $@;\
-	 fi
+	@if test -f $@; then /bin/rm $@; fi
+	@ln -f bin/wbinfo@EXEEXT@ $@
 
 bin/lwimsg@EXEEXT@: bin/smbcontrol@EXEEXT@
-	@if test ! -f $@; then \
-		ln bin/smbcontrol@EXEEXT@ $@;\
-	 fi
+	@if test -f $@; then /bin/rm $@; fi
+	@ln -f bin/smbcontrol@EXEEXT@ $@
 
 IDMAP_LWOPEN_OBJ = winbindd/lwopen/idmap_lwopen.o \
 		   winbindd/lwopen/mapfile.o
