Index: likewise-open/samba/source/winbindd/winbindd_group.c
===================================================================
--- likewise-open.orig/samba/source/winbindd/winbindd_group.c	2008-08-12 12:50:13.000000000 -0500
+++ likewise-open/samba/source/winbindd/winbindd_group.c	2008-08-12 15:37:54.000000000 -0500
@@ -596,7 +596,7 @@
 		/* If we have no more groups to expand, break out
 		   early */
 
-		if ( !&new_glist )
+		if (new_glist == NULL)
 			break;
 
 		/* One more round */
